graylog2-web-interface/src/views/stores/ViewStore.js

```diff
@@ -27,30 +27,26 @@ export type ViewStoreState = {
 
 type ViewActionsType = RefluxActions<{
   create: (ViewType, ?string) => Promise<ViewStoreState>,
-  description: (string) => Promise<ViewStoreState>,
   load: (View, ?boolean) => Promise<ViewStoreState>,
   properties: (Properties) => Promise<void>,
   search: (Search) => Promise<View>,
   selectQuery: (string) => Promise<string>,
   state: (ViewState) => Promise<View>,
-  summary: (string) => Promise<void>,
-  title: (string) => Promise<void>,
+  update: (View) => Promise<ViewStoreState>,
```

Comment:
I am not sure if this action should return the `ViewStoreState` or just the `View`. `ViewManagementActions.update` returns only the view, but I decided to return the `ViewStoreState`, to unify the type of the return value with `ViewActions.create``